conveyor/config.py

```diff
@@ -121,4 +122,15 @@ def configure():
         documentation_top,
     )
 
+    # Allow cross-origin GETs by default
+    cors = aiohttp_cors.setup(app, defaults={
+            "*": aiohttp_cors.ResourceOptions(
+                allow_methods="GET",
+            )
+    })
+
+    # Add the default CORS configuration to all routes
+    for route in list(app.router.routes()):
+        cors.add(route)
```

Comment:
I think I'd rather see this applied only to the `/packages` routes? We don't want to do this for legacy docs as well do we?

Reply:
+1, forgot about the docs routes.

Reply:
cool. just updated it. i have it working on the subroutes on `/packages`. is that overkill or does applying to the package level push it down downstream?

Reply:
that looks spot on, I believe each route needs to be CORS'd independently.